app/src/main/kotlin/org/fossify/keyboard/extensions/Context.kt

```diff
@@ -204,6 +205,7 @@ fun Context.getKeyboardLanguageText(language: Int): String {
         LANGUAGE_FRENCH_BEPO -> "${getString(R.string.translation_french)} (BEPO)"
         LANGUAGE_GERMAN -> getString(R.string.translation_german)
         LANGUAGE_GREEK -> getString(R.string.translation_greek)
+        LANGUAGE_GUJARATI -> getString(R.string.translation_gujarati)
```

Comment:
This string doesn't exist. You should add it in Commons in https://github.com/FossifyOrg/Commons/blob/master/commons/src/main/res/values/strings.xml. When creating a pull request in Commons, please mention that it's for this one.

Reply:
It's ok now.